packages/datagateway-download/package.json

```diff
@@ -20,8 +21,12 @@
     "react": "^16.13.1",
     "react-dom": "^16.11.0",
     "react-i18next": "^11.7.0",
+    "react-redux": "^7.1.0",
```

Comment:
Why does `datagateway-download` now require redux? This adds a massive overhead. The loading state in `datagateway-common` will not indicate anything useful here I think. If a loading indicator is needed in download, then an alternative solution needs to be thought of (e.g. do it on the individual table components, so that the loading boolean can be passed in that way)

Reply:
Have moved the loading bars from `downloadTab` to `downloadCartTable` and `downloadStatusTable`, where `dataloaded` is used to perform the logic. This removes the need for redux to access `loading` (as we do in search and dataview).